Duplicati/Library/Modules/Builtin/run-script-example.sh

```diff
@@ -142,7 +142,7 @@ then
 		if [ "$DUPLICATI__volsize" == "" ] || ["$DUPLICATI__volsize" == "50mb"]
```

Comment:
Are these environment variables also incorrect?

Reply:
No, they are not correct, compare [RunScript.cs, Line 214](https://github.com/duplicati/duplicati/blob/master/Duplicati/Library/Modules/Builtin/RunScript.cs#L214). The environment variable should be at run-script-example.sh
```suggestion
		if [ "$DUPLICATI__dblock_size" == "" ] || ["$DUPLICATI__dblock_size" == "50mb"]
```
at run-script-example.bat
```suggestion
IF "%DUPLICATI__dblock_size%" == "" GOTO SET_VOLSIZE
IF "%DUPLICATI__dblock_size%" == "100MB" GOTO SET_VOLSIZE
```
Should I add a secound commit and pull request with this changes?

Reply:
I changed the environment variable in my last commit 2ef30fe.